### PR TITLE
Invoke angry_purple_tiger from CLI

### PR DESCRIPTION
**Pull request by vihu**
- Add bin support
- Add clap dependency

### CODE REVIEW

File: src/bin.rs [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#[macro_use]
```
Comment (madninja) [inline src/bin.rs]: Do you still have use this? 

I think you can now just use `clap::clapp_app;`

File: Cargo.toml [hunk 2]
```diff
@@ -8,7 +8,13 @@ description = "animal-based hash digests for humans"
...
 edition = "2018"
 
+
+[[bin]]
+name = "angry_purple_tiger"
+path = "src/bin.rs"
+
 [dependencies]
 md5 = "0"
+clap = "2.33.3"
```
Comment (madninja) [inline Cargo.toml]: make broader.. just spec `clap = "2"`

**Review by madninja — CHANGES_REQUESTED** [review]

File: src/bin.rs (hunk 1 above)
Comment (vihu) [reply src/bin.rs]: ah yes, good call. updated 646714d

File: Cargo.toml (hunk 2 above)
Comment (vihu) [reply Cargo.toml]: ack, updated in 646714d

**Review by madninja — APPROVED** [review]

**Pull request merged**